misc/pytorch_toolkit/chest_xray_screening/chest_xray_screening/utils/generate.py

```diff
@@ -0,0 +1,87 @@
+import torchvision.models.densenet as modelzoo
+
+MIN_CHANNELS = 32 # The minimum number of channels at the first convolution
+CHANNEL_FACTOR = 8
+def get_size(blocks, growth_rate, initial_channels):
+
+    depth = 5 + 2 * sum(blocks)
+    # DenseNet architecture (irrespective of its size 121 or 169 etc) has 5 layers
+    # (an input conv layer+final classification layer+3 1X1 conv in the transition layers)
+    # Apart from that, there are 4 Dense Blocks.
+    # Each Dense Block repeats a Basic Block inside it FOR L_i times (ie ith Block uses li blocks )
+    # and each basic block has 2 convolutional layers.
+    l1, l2, l3, l4 = blocks
+    width = growth_rate * l4 + (
+        growth_rate * l3 + (
+            growth_rate * l2 + (
+                growth_rate * l1 + initial_channels) // 2) // 2) // 2
+
+    return depth, int(width)
+
+
+def new_lengths(alpha, l1=6, l2=12, l3=24, l4=16):
+    l1 = int(alpha * l1)
+    l2 = int(alpha * l2)
+    l3 = int(alpha * l3)
+    l4 = int(alpha * l4)
+
+    depth = 5 + 2 * (l1 + l2 + l3 + l4)
+    # DenseNet architecture (irrespective of its size 121 or 169 etc) has 5 layers
+    # (an input conv layer+final classification layer+3 1X1 conv in the transition layers)
+    # Apart from that, there are 4 Dense Blocks.
+    # Each Dense Block repeats a Basic Block inside it FOR L_i times (ie ith Block uses li blocks )
+    # and each basic block has 2 convolutional layers.
+
+#     l1 += int((121*alpha - depth)/2) # Uncomment for more accurate depth
+#     depth = 5+2*(l1+l2+l3+l4) # Uncomment with the previous line
+
+    return depth, [l1, l2, l3, l4]
+
+
+def new_channels(block_new, alpha, beta, growth_rate=32,
+                 init_chan=64, bn_size=4):
+    l1, l2, l3, l4 = block_new
+    growth_rate = round(beta / alpha * growth_rate)
+    gw = growth_rate
+    bn_size = round(alpha * bn_size)
+    if bn_size < 2: # The bn_size should be atleast 1

```

Comment:
```suggestion
    if bn_size <= 1: # The bn_size should be atleast 1

```
Though it is the same in this case, such if condition corresponds to its comment more ;)